daily-coding-challenges/24-06-2020/repeatString.js

```diff
@@ -13,7 +13,10 @@
 
 const repeatStringNumTimes = (str, num) => {
 	/* Your implementation goes here */
-	return str;
+	if(num > 0) 
```

Comment:
@iamfortune, What happens if `num` is equal to 0. 
For example if I do `repeatStringNumTimes('abc', 0)`, what should it return? There is one more case you're missing here

Reply:
and edge case... hmm

Reply:
solved it